bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/handler/DaikinAcUnitHandler.java

```diff
@@ -202,7 +205,21 @@ protected void changeMode(String mode) throws DaikinCommunicationException {
         }
         ControlInfo info = webTargets.getControlInfo();
         info.mode = newMode;
-        webTargets.setControlInfo(info);
+        if (autoModeValue.isPresent()) {
+            info.autoModeValue = autoModeValue.get();
+        }
+        boolean accepted = webTargets.setControlInfo(info);
+
+        // If mode=0 is not accepted try AUTO1 (mode=1)
```

Comment:
Just double check question: 
As the Mode enum also has:
```
 AUTO7(7), // Some adapters may return 7 as auto (heating) 
```
and it is normalised to `Mode.AUTO` just like `Mode.AUTO1` it looks like it is missing from this addition.

Reply:
Yes, this is deliberate. From what we can gather:

- The current library uses mode=0, and that's why it remains the current default for AUTO
- We "suspect" that all models accept mode=1 (not fully tested against all supported models). A [post was made on the forum](https://community.openhab.org/t/daikin-binding-test-of-auto-mode-value-we-need-your-help/148919) to get others to test this. In the mean time, mode=0 remains the default for AUTO.
- BRP069A81 (for which this PR is intended), _only_ accepts mode=1. It does not accept mode=0 or mode=7
- Models that accept mode=0 "seem to" also accept mode=1 and mode=7 (not fully tested against all supported models)
- We "believe" that mode=0 and mode=7 have different meanings, and mode=1 is actually the most "AUTO" amongst the three codes. 
- There are no models, to our knowledge, that only accept mode=7 but not mode=0 or mode=1, so trying mode=7 is currently not necessary nor requested.

So in short:
- Use mode=0 as it was before, but if it doesn't work:
- Try mode=1 because that's the only one recognised by BRP069A81 (this PR)
- mode=7 is not attempted because BRP069A81 doesn't recognise it.

See #15375

Reply:
Thanks for the extensive clarification.